clusters/app.ci/supplemental-ci-images/html-proofer_is.yaml

```diff
@@ -31,3 +33,29 @@ spec:
     dockerStrategy:
       imageOptimizationPolicy: SkipLayers
     type: Docker
+---
+kind: BuildConfig
```

Comment:
Since we are in the effort of getting rid of all buildconfigs that we can't control the multi-architecture, do you think that this image is going to be used to build something on ci-tools? The ci-tools job is already running on build10. I didn't check, but if something is using this image this PR will break the ci-tools postsubmit job.

Reply:
It not getting used, it is a test image for QCI.